bin/prettier.js

```diff
@@ -43,7 +43,7 @@ if (argv["version"]) {
 
 const filepatterns = argv["_"];
 const write = argv["write"];
-const stdin = argv["stdin"];
+const stdin = argv["stdin"] || (!filepatterns.length && !process.stdin.isTTY);
```

Comment:
What happens if you just do

```js
./bin/prettier
```

does it still prints the warning, or just tries to pretty print an empty string?